Sources/Polyline/Polyline.swift

```diff
@@ -51,11 +55,13 @@ public struct Polyline {
     public let encodedLevels: String?
     
     /// The array of location (computed from coordinates)
+    #if canImport(CoreLocation)
     public var locations: [CLLocation]? {
         return self.coordinates.map(toLocations)
     }
+    #endif
     
-    #if !os(watchOS)
+    #if canImport(MapKit)
```

Comment:
On watchOS, MapKit can be imported, but its symbols are marked as unavailable, triggering the following compiler error:

```
/path/to/Polyline/Sources/Polyline/Polyline.swift:67:28: 'MKPolyline' is unavailable in watchOS
/MapKit.MKPolyline:2:12: 'MKPolyline' has been explicitly marked unavailable here
```

Reply:
Fixed in #67.